org.lflang.tests/src/org/lflang/tests/cli/LfcCliTest.java

```diff
@@ -98,17 +104,71 @@ public void testGenInSrcDir(@TempDir Path tempDir) throws IOException {
                     .check("bin", isDirectory())
                     .check("src-gen/File/File.py", isRegularFile());
             });
+    }
 
+    @Test
+    public void testBuildParams(@TempDir Path tempDir)
```

Comment:
Another test vector that I think we should have is a set of known-bad inputs and the handling of those.

Reply:
Done!